docs/getting-started/logcli.md

```diff
@@ -72,51 +72,127 @@ A command-line for loki.
 
```

Comment:
Sorry this is as high up as I can comment, but can we change the syntax highlighter for this from bash and either remove it or use `nohighlight` or something that doesn't end up randomly highlighting key words